samples/csharp_dotnetcore/51.teams-messaging-extensions-action/Bots/TeamsMessagingExtensionsActionBot.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
This warning may be fixed by returning `await Task.FromResult(new MessagingExtensionActionResponse());` (We should try it)

Reply:
Done. Thanks!